main.go

```diff
@@ -599,6 +603,16 @@ func (c *config) offsetSelection(file ast.Node) (int, int, error) {
 	return start, end, nil
 }
 
+func isPublicName(name string) bool {
+	for _, c := range name {
+		if unicode.IsUpper(c) {
+			return true
+		}
+		return false
+	}
+	return false
+}
```

Comment:
I think you can simplify this with the following function:

```go
func isUpper(name string) bool {
	for _, c := range name {
		return unicode.IsUpper(c)
	}
	return false
}
```

example: https://play.golang.org/p/J2SjN9V3Dhh

Reply:
Oh, such an obvious thing))) Fixed of course. Thank you!